projects/fsastorefrontlib/src/assets/translations/de/changeRequest.de.ts

```diff
@@ -1,18 +1,18 @@
 export const changeRequest = {
   changeRequest: {
-    false: '[DE] Not Included',
-    true: '[DE] Included',
-    description: '[DE] Description',
-    currentSelection: '[DE] Current Selection',
-    newSelection: '[DE] New Selection',
-    currentPremium: '[DE] Current Premium',
-    newPremium: '[DE] New Premium',
+    false: 'Nicht enthalten',
+    true: 'Beinhaltet',
+    description: 'Beschreibung',
+    currentSelection: 'Aktuelle Auswahl',
+    newSelection: 'Neue Auswahl',
+    currentPremium: 'Aktuelle Prämie',
+    newPremium: 'Neue Prämie',
     confirmation: {
-      title: '[DE] Change Request Confirmation',
+      title: 'Bestätigung der Antragsänderung',
       message:
-        '[DE] Your change has been successfully submitted. You can find processing email in the',
-      inbox: '[DE] Inbox',
+        'Ihre Änderunge wurde erfolgreich übermittelt. Die Verarbeitung von E-Mails finden Sie in der',
+      inbox: 'Posteingang',
     },
-    autoInformation: '[DE] Auto Information',
+    autoInformation: 'Automatische Informationen',
```

Comment:
I'm not sure this translation refers to car information, rather to automatic information... Are we sure this is the right translation?

Reply:
Probably not correct. Added as "Fahrzeug Informationen".